org.mwc.debrief.lite/build.xml

```diff
@@ -1,109 +1,146 @@
+		<and>
+			<os family="mac"/>
+			<available file="${java.home}/../Classes" type="dir"/>
+		</and>
+	</condition>
+	<property name="dir_bootclasspath" value="${java.home}/lib"/>
+	<path id="path_bootclasspath">
+		<fileset dir="${dir_bootclasspath}">
+			<include name="*.jar"/>
+		</fileset>
+	</path>
+	<property name="bootclasspath" refid="path_bootclasspath"/>
+	<condition property="bundleBootClasspath" value="${JavaSE-1.7}"	>
+		<isset property="JavaSE-1.7"/>
+	</condition>
+	<property name="bundleBootClasspath" value="${bootclasspath}"/>
+	<xmlproperty file="pom.xml"/>
+		
+	<target name="init">
+		<condition property="pluginTemp" value="${buildTempFolder}/plugins"		>
+			<isset property="buildTempFolder"/>
+		</condition>
+		<property name="pluginTemp" value="${basedir}"/>
+		<condition property="build.result.folder" value="${pluginTemp}/org.mwc.debrief.lite_1.0.377"		>
+			<isset property="buildTempFolder"/>
+		</condition>
+		<property name="build.result.folder" value="${basedir}"/>
+		<property name="temp.folder" value="${basedir}/temp.folder"/>
+		<property name="dist.dir" value="dist"/>
+		<property name="dist.lib.dir" value="dist/lib"/>
+		<property name="product.version" value="${project.version}"/>
+	</target>
+
+	<target name="debrief_lite.jar" depends="init" unless="debrief_lite.jar" description="Create jar: org.mwc.debrief.lite debrief_lite.jar.">
+		<delete dir="${temp.folder}/debrief_lite.jar.bin"/>
+		<mkdir dir="${temp.folder}/debrief_lite.jar.bin"/>
+		<path id="debrief_lite.jar.classpath">
+			<pathelement location="cmap_legacy.jar"/>
+			<pathelement location="debrief_legacy.jar"/>
+			<fileset dir="../org.mwc.cmap.legacy/libs" includes="*.jar"></fileset>
+			<fileset dir="../org.mwc.debrief.legacy/libs" includes="*.jar"></fileset>
+			<fileset dir="../org.mwc.cmap.jfreechart/libs" includes="*.jar"></fileset>
+			<fileset dir="../org.geotools/libs" includes="*.jar"></fileset>
+			<!--fileset dir="libs" includes="*.jar"></fileset-->
+		</path>
+		<!-- compile the source code -->
+		<javac destdir="${temp.folder}/debrief_lite.jar.bin" failonerror="true">
+			<classpath refid="debrief_lite.jar.classpath" />
+			<src path="src/"			/>
+		</javac>
+		<!-- Copy necessary resources -->
+		<copy todir="${temp.folder}/debrief_lite.jar.bin" failonerror="true" overwrite="false">
+			<fileset dir="src/main/java">
+				<exclude name="**/*.java"/>
+				<exclude name="**/package.htm*"/>
+			</fileset>
+			<fileset dir="src/main/java">
+				<include name="images"/>
+		 	</fileset>
+		</copy>
+		<jar destfile="${temp.folder}/debrief_lite.jar" basedir="${temp.folder}/debrief_lite.jar.bin">
+			<manifest>
+				<attribute name="Main-Class" value="org.mwc.debrief.lite.DebriefLiteApp"/>
+				<attribute name="Implementation-Title" value="Debrief Lite"/>
+				<attribute name="Implementation-Version" value="${product.version}"/>
+				<attribute name="Implementation-Vendor" value="DeepBlue C Ltd"/>
+				<attribute name="Class-Path" 

```

Comment:
@ayeshare - we're pulling in eclipse SWT libraries in this classpath.  I'm really hoping we can do without them. 

Separate to that, if possible I'd like to avoid this "magic string" of jar names.  I presume we already define a .classpath file somewhere for Debrief-Lite. Can we pull in the contents of that file into this variable?

(I'm hoping that by managing the dependencies from something like .classpath, Eclipse will help us with compiler warnings and/or refactorings)

Reply:
If you are referring to the libs in .classpath file: the .classpath file is only used by developers in the workbench.
I really wish we could remove the magic string of jar names.  One jar was doing that for us.  We can put that in a manifest file and include the manifest file, which is the other option.

Reply:
We're speaking beyond what I really understand, but I like the sound of the manifest file. Could you try it?